tests/models/jit_test.py

```diff
@@ -0,0 +1,105 @@
+import torch
+import pytest
+from torch.testing import assert_allclose
+from asteroid.models import DeMask, ConvTasNet, DPRNNTasNet, DPTNet, LSTMTasNet
+
+
+@pytest.fixture(scope="module")
+def small_model_params():
+    params = {
+        ConvTasNet.__name__: {
+            "n_src": 2,
+            "n_repeats": 2,
+            "n_blocks": 3,
+            "bn_chan": 16,
+            "hid_chan": 4,
+            "skip_chan": 8,
+            "n_filters": 32,
+        },
+        DPRNNTasNet.__name__: {
+            "n_src": 2,
+            "n_repeats": 2,
+            "bn_chan": 16,
+            "hid_size": 4,
+            "chunk_size": 20,
+            "n_filters": 32,
+        },
+        DPTNet.__name__: {
+            "n_src": 2,
+            "ff_hid": 10,
+            "chunk_size": 4,
+            "n_repeats": 2,
+        },
+        LSTMTasNet.__name__: {
+            "n_src": 2,
+            "hid_size": 4,
+            "n_layers": 1,
+            "n_filters": 32,
+            "dropout": 0.0,
+        },
+        DeMask.__name__: {
+            "input_type": "mag",
+            "output_type": "mag",
+            "hidden_dims": [64],
+            "dropout": 0,
+            "activation": "relu",
+            "mask_act": "relu",
+            "norm_type": "gLN",
+            "n_filters": 8,
+            "stride": 4,
+            "kernel_size": 8,
+        },
+    }
+
+    return params
+
+
+def test_enhancement_model(small_model_params):
+    params = small_model_params["DeMask"]
+    filter_banks = ["free", "stft", "analytic_free", "param_sinc"]
+    device = get_default_device()
+    inputs = ((torch.rand(1, 1000, device=device) - 0.5) * 2,)
+    test_data = torch.rand(1, 32, device=device)
```

Comment:
Why from 1000 to 32?

Reply:
Just to avoid using the same length at the input but does the value really matter?

Reply:
32 samples makes 2 frames, it seems small. But I agree to have samples with different sizes.